src/command/output.rs

```diff
@@ -23,6 +26,18 @@ pub enum RoverStdout {
 impl RoverStdout {
     pub fn print(&self) {
         match self {
+            RoverStdout::DocsList(shortlinks) => {
+                eprintln!(
+                    "You can open any of these documentation pages by running {}.\n",
+                    Yellow.normal().paint("rover docs open <slug>")
+                );
+                let mut table = Table::new();
+                table.add_row(row!["Slug", "Description"]);
```

Comment:
An aesthetics suggestion: how do you feel about having the header's row be bold?

Reply:
seems good!